src/app/webbrowser/SettingsPage.qml

```diff
@@ -129,6 +129,26 @@ FocusScope {
                     }
                 }
 
+                ListItem {
+                    objectName: "autoFitToWidthEnabled"
+
+                    ListItemLayout {
+                        title.text: i18n.tr("Automatic Fit to Width")
+                        subtitle.text: i18n.tr("Zooms to body.scrollWidth after domain changed, if no specific zoom is saved.")
```

Comment:
This flows outside of the page, but also doesn't make much sense to and end-user... Maybe something more like "Zoom out to show more content on some websites." @cibersheep, language thoughts?

Reply:
Mmm... what about?

"Adjust the width of the website to the window"

Reply:
Description was changed